src/pages/Swap/index.tsx

```diff
@@ -140,7 +141,7 @@ export default function Swap({ history }: RouteComponentProps) {
     [independentField, parsedAmount, showWrap, trade]
   )
 
-  const fiatValueInput = useUSDCValue(parsedAmounts[Field.INPUT])
+  const fiatValueInput = useUSDCValue(parsedAmounts[Field.INPUT] ?? pricedAmount)
```

Comment:
did you consider using useUSDCPrice instead?

useUSDCValue calls useUSDCPrice with the currency. I feel like providing pricedAmount in the hook complicates it a bit.

Reply:
I went this way because all the child components rely on a `CurrencyAmount<Token>` type and `useUSDCPrice` returns `Price<Currency, Token>`.  `pricedAmount` is a unitary amount of the currency, considered it to be the simplest way to use existing interfaces to achieve the display of a fiat value in the `FiatRow` of a `CurrencyInputPanel`.  Thoughts?

Reply:
essentially, I'm passing `useUSDCValue` either the `parsedAmount` from the input field, or a unitary currency amount (i.e., `pricedAmount`) for the input currency amount (e.g., 1 ETH if there is no parsed amount and ETH is the input token).